code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -296,55 +238,17 @@
 
 		var/obj/item/organ/external/E = bodyparts_by_name[organ_tag]
 		if(!E)
-			wound_flavor_text["[organ_tag]"] = "<span class='warning'><b>[t_He] [t_is] missing [t_his] [organ_descriptor].</b></span>\n"
-		else
-			continue
-
-	for(var/obj/item/organ/external/temp in bodyparts)
-		if(temp.status & ORGAN_ROBOT)
-			if(!(temp.brute_dam + temp.burn_dam))
-				if(!isSynthetic())
-					wound_flavor_text["[temp.limb_name]"] = "<span class='warning'>[t_He] [t_has] a robotic [temp.name]!</span>\n"
-					continue
-			else
-				wound_flavor_text["[temp.limb_name]"] = "<span class='warning'>[t_He] [t_has] a robotic [temp.name]. It has"
-			if(temp.brute_dam)
-				switch(temp.brute_dam)
-					if(0 to 20)
-						wound_flavor_text["[temp.limb_name]"] += " some dents"
-					if(21 to INFINITY)
-						wound_flavor_text["[temp.limb_name]"] += pick(" a lot of dents"," severe denting")
-			if(temp.brute_dam && temp.burn_dam)
-				wound_flavor_text["[temp.limb_name]"] += " and"
-			if(temp.burn_dam)
-				switch(temp.burn_dam)
-					if(0 to 20)
-						wound_flavor_text["[temp.limb_name]"] += " some burns"
-					if(21 to INFINITY)
-						wound_flavor_text["[temp.limb_name]"] += pick(" a lot of burns"," severe melting")
-			if(wound_flavor_text["[temp.limb_name]"])
-				wound_flavor_text["[temp.limb_name]"] += "!</span>\n"
+			wound_flavor_text["[organ_tag]"] = "<b>[t_He] [t_is] missing [t_his] [organ_descriptor].</b>\n"

```

Comment:
you're inconsistent with `<b>` and `<B>` again down lower down... can we pick just one version and always use that?